### ISSUE TITLE
support ROS-I development status / maturity

### ISSUE THREAD

**Issue by jbohren** [rosindex/rosindex.github.io#102]
@shaun-edwards I noticed that ROS-I packages have this additional software status tag on the wiki

http://wiki.ros.org/Industrial/Software_Status

Do you think this could be put in an `<export>` tag like:

```xml
<export>
  <industrial>
    <maturity status="experimental"/>
  </industrial>
</export>
```

This has the added benefit of having authorship accountability, since it will be in version-controlled source and not just displayed on a wiki page.

**Comment by shaun-edwards** [comment]
It would be great if every package has such a status.  I tried a while back to get "universal" ROS adoption, but several important people weren't on board.  Other [projects](http://wiki.ros.org/rtmros_nextage) have used the software status.

My one concern would be keeping wiki material and the rosindex material synced.  This is probably a general concern inherent in making  a switch like this.  Probably more appropriate for a different discussion and larger group.

+1 to the idea itself.  I assume the `<export>` tag would go in the `package.xml`?

I would recommend using xml elements vs. attributes (see [here](http://www.w3schools.com/dtd/dtd_el_vs_attr.asp) for discussion).

**Comment by jbohren** [comment]
> It would be great if every package has such a status. I tried a while back to get "universal" ROS adoption, but several important people weren't on board. Other projects have used the software status.

Yeah, there used to be the `review` status, but that went away along with reviews.

> My one concern would be keeping wiki material and the rosindex material synced. This is probably a general concern inherent in making a switch like this. Probably more appropriate for a different discussion and larger group.

I'm not really concerned about keeping the material "synced" since nothing is actually stored in rosindex. All rosindex does is display what it finds in the repositories. If you move this information to the package.xml, I'm sure you could work with the OSRF guys to get a macro which displays the value extracted from that, too.

> +1 to the idea itself. I assume the <export> tag would go in the package.xml?

Yeah.

> I would recommend using xml elements vs. attributes (see here for discussion).

Yeah, I've heard such arguments before. I personally don't care about which convention it uses.
